app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -168,6 +169,10 @@
 
   StaticAnalytics.prototype.setThemesDimension = function(themes) {
     this.setDimension(3, themes || 'other');
+  }
+
+  StaticAnalytics.prototype.setContentIDDimension = function(contentID) {
```

Comment:
I think this should be `setContentIdDimension`. See this SO post on acronyms in camelcase http://stackoverflow.com/questions/15526107/acronyms-in-camelcase